docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -125,25 +125,21 @@ const theme = createTheme({
 
 The data grid comes with two different licenses:
```

Comment:
```suggestion
The data grid comes in two different versions:
```

Reply:
I disagree with this change, I think that what matters more is the license, not the package

Reply:
It's confusing. It sounds like one package with two licenses.

```suggestion
The data grid comes in two separately licensed packages:
```

Reply:
Ok, I have tried something bolder.